chart/templates/connection-secret-job.yaml

```diff
@@ -1,5 +1,10 @@
 {{- $db := index .Values "timescaledb-single" -}}
-{{- if and $db.enabled (eq .Values.promscale.connectionSecretName "") (eq .Values.promscale.connection.password "") (.Release.IsInstall) -}}
+{{- if and
+    $db.enabled
+    (eq .Values.promscale.connectionSecretName "tobs-promscale-connection")
+    (eq .Values.promscale.connection.password "")
+    (eq .Values.promscale.connection.uri "")
```

Comment:
Why keep `.Values.promscale.connection.uri` and `.Values.promscale.connection.password` here if you are removing them from `values.yaml`?

Reply:
`.Values.promscale.connection` object is never removed as it is set in promscale helm chart, in tobs we are just removing the override. And since the logic created in this PR shouldn't be active when the user sets the values below, we still need checks for `password` and for `uri`.
```yaml
promscale:
  connectionSecretName: ""
  connection:
    uri: XYZ
```
or
```yaml
promscale:
  connectionSecretName: ""
  connection:
    ...
    password: XYZ
```